docs/src/blockchain-development/hashing_and_cryptography.md

```diff
@@ -0,0 +1,15 @@
+# Calling Contracts
```

Comment:
Title

Reply:
I was getting pretty tired with this one... Damn.